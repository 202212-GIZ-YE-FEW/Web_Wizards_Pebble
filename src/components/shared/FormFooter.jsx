import Link from "next/link";

const FormFooter = ({ message, linkText, linkHref }) => {
    return (
        <div className='pb-3 px-3'>
            <p>
                {message}
                {linkHref && (
                    <Link href={linkHref}>
                        <span className='text-gray-800 underline'>
                            {linkText}
                        </span>
                    </Link>
                )}
            </p>
        </div>
    );
};
export default FormFooter;
