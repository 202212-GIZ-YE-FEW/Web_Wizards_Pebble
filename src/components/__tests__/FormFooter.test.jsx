import renderer from "react-test-renderer";

import FormFooter from "@/components/FormFooter";

it("renders correctly", () => {
    const tree = renderer.create(<FormFooter />).toJSON();
    expect(tree).toMatchSnapshot();
});
