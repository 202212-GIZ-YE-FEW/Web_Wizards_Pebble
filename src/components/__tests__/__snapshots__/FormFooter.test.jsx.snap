// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="pb-3 px-3"
>
  <p />
  <button
    className="button__ButtonWrapper-sc-1aw98b4-0 button !rounded-none !text-black-100 !bg-transparent !shadow-none !border-none after:content-[url(\\"/arrow.svg\\")] after:mt-[1px] ltr:after:ml-4 after:absolute rtl:after:rotate-180 rtl:after:mr-4"
    disabled={false}
  >
    forgetPassword
  </button>
</div>
`;
